app/views/users/show.html.slim

```diff
@@ -0,0 +1,3 @@
+h4 align="center" E-mail: #{@user.email}
+.btn.btn-outline-success id='edit_profile_button'
```

Comment:
I heard an idea that it's better to apply CSS only with classes. And use id's for JavaScript. That give you and your teem a confidence that in case of deleting some CSS / JS code you need also to delete only one class / id in your view file. And you don't need to search for a lot of dependencies in our front end.

So, let's follow the same strategy. Change id to class, please